Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM openjdk:8u171-jre-alpine
 
-ARG kafka_version=1.1.0
+ARG kafka_version=1.1.1
```

Comment:
If this Dockerfile uses Kafka 1.1.1 with Scala 2.12 (which is recommended) why does `.travis.yml` use 2.11? Just curious.

Reply:
http://kafka.apache.org/downloads recommends 2.11 with version 1.1.1. Is 2.12 recommended use documented anywhere else? 

Travis ci always passes build arguments, so the default values should probably be removed as they are redundant.

Reply:
Sorry for the "noise". I thought that 2.12 was the recommended version.

Reply:
no problem. Good to have another pair of eyes. And it highlights the fact that we should probably remove the default values. Will address as part of https://github.com/wurstmeister/kafka-docker/issues/363